### ISSUE TITLE
Installation issue with 0.24.5 iso's

### ISSUE THREAD

**Issue by ghost** [AerynOS/discussions#17]
Title: Installation issue with 0.24.5 iso's

Description: when installing either one of the two iso's the installation halts with an error-loop. The onscreen errormessage is as follows:

> Entering emergency mode. Exit the shell to continue.
> /bin/dracut/emergency:: line 37: exec: sulogin: not found
> dracut-warning: can't mount root filesystem
> /dev/disk/by-label/SERPENTISO does not exist
> /dev/root does not exist

- Checked with both cosmic and gnome iso; 
- Tried with two different usb sticks, both worked today with multiple other images;
- Used both dd (following the instruction on the site) and Impression (they show up as SERPENTISO afterwards);
- Tried Ventoy as well, with the same result, secure boot is indeed disabled (otherwise Ventoy wouldn't work)
- Mothership: KDE 6.24 on Slackware Current

**Comment by chrisdebian** [comment]
@thesaigoneer Is this still an issue with the latest [release](https://serpentos.com/download/); it solved a few problems for me.

Chris

**Comment by ghost** [comment]
@chrisdebian  yes, unfortunately I didn't manage to get it installed; gave it a retry last night before answering. Followed the specific partitioning advice from the forums as well. No biggie; I'll stick around for later versions. To verify all I installed Chimera Linux and later my version of Aurora; both worked. That's what alpha's are for ;-) Closing the issue for now as well, as Arnie said: I'll be back.

**Comment by chrisdebian** [comment]
Cheers,  @thesaigoneer 

I tried Chimera, very promising.

Chris
